docs/source/contributing/developing-core.md

```diff
@@ -104,16 +104,16 @@ For the fish shell, see [nvm.fish](https://github.com/foldmap/nvm.fish).
 1.  Install or update the supported LTS versions of Node.js, then activate the version supported in Volto.
 
     ```shell
-    nvm install 21.5.0
-    nvm use 21.5.0
+    nvm install "lts/*"
+    nvm use 20
     ```
 
 2.  Verify that the supported version of Node.js is activated.
 
     ```shell
-    node -v
+    node -verion
```

Comment:
This is nonsense. You must verify that commands you document actually work. Revert.